src/MCPServer/install/README.md

```diff
@@ -1,13 +1,135 @@
+by precedence (with the last listed item winning prioritization):
+
+- Application defaults
+- Configuration files: (see [`the example`](./serverConfig.conf))
+    - `/etc/archivematica/archivematicaCommon/dbsettings` (optional)
+    - `/etc/archivematica/MCPServer/serverConfig.conf` (optional)
+- Environment variables (always win precedence)
+
+## Environment variables
+
+The value of an environment variable is a string of characters. The
+configuration system coerces the value to the types supported:
+
+- `string` (e.g. `"foobar"`)
+- `int` (e.g. `"60"`)
+- `float` (e.g. `"1.20"`)
+- `boolean` where truth values can be represented as follows:
+    - True (enabled):  `"1"`, `"yes"`, `"true"` or `"on"`
+    - False (disabled): `"0"`, `"no"`, `"false"` or `"off"`
+
+Certain environment strings are mandatory, i.e. they don't have defaults and
+the application will refuse to start if the user does not give them one.
+
+Please be aware that Archivematica supports different types of distributions
+(Ubuntu/CentOS packages, Ansible or Docker images) and they may override some
+of these settings or provide values to mandatory fields.
+
+This is the full list of environment strings supported:
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_DJANGO_SECRET_KEY`**:
+    - **Description:** a secret key used for cryptographic signing. See [SECRET_KEY](https://docs.djangoproject.com/en/1.8/ref/settings/#secret-key) for more details.
+    - **Type:** `string`
+    - :red_circle: **Mandatory!**
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_SHAREDDIRECTORY`**:
+    - **Description:** location of the Archivematica Shared Directory.
+    - **Type:** `string`
+    - **Default:** `"/var/archivematica/sharedDirectory/"`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_PROCESSINGXMLFILE`**:
+    - **Description:** name of the processing configuration file.
+    - **Type:** `string`
+    - **Default:** `"processingMCP.xml"`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_MCPARCHIVEMATICASERVER`**:
+    - **Description:** address of the Gearman server.
+    - **Type:** `string`
+    - **Default:** `"localhost:4730"`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_WATCHDIRECTORYPATH`**:
+    - **Description:** location of the Archivematica Watched Directory.
+    - **Type:** `string`
+    - **Default:** `"/var/archivematica/sharedDirectory/watchedDirectories/"`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_PROCESSINGDIRECTORY`**:
+    - **Description:** loation of the processing directory.
+    - **Type:** `string`
+    - **Default:** `"/var/archivematica/sharedDirectory/currentlyProcessing/"`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_REJECTEDDIRECTORY`**:
+    - **Description:** location of the rejected directory.
+    - **Type:** `string`
+    - **Default:** `"%%sharedPath%%rejected/"`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_WAITONAUTOAPPROVE`**:
+    - **Description:** suspend execution of the current thread before attempting to approve a chain when it is pre-configured.
+    - **Type:** `int`
```

Comment:
Could this be changed to a boolean?

Reply:
Good idea sir!

Reply:
Actually not really, it's used in a time.sleep to suspend the threads. Will update the description to reflect that.